scipy/fft/_helper.py

```diff
@@ -37,22 +37,22 @@ def next_fast_len(target, real=False):
 
     Examples
     --------
-    On a particular machine, an FFT of prime length takes 17 ms:
+    On a particular machine, an FFT of prime length takes 11.4 ms:
 
     >>> from scipy import fft
     >>> min_len = 93059  # prime length is worst case for speed
     >>> a = np.random.randn(min_len)
     >>> b = fft.fft(a)
 
     Zero-padding to the next regular length reduces computation time to
-    1.3 ms, a speedup of 13 times:
+    1.6 ms, a speedup of 7.3 times:
 
-    >>> fft.next_fast_len(min_len)
+    >>> fft.next_fast_len(min_len, True)
```

Comment:
Keyword arguments should be fixed now:
```suggestion
    >>> fft.next_fast_len(min_len, real=True)
```